formicarium/formicarium/Lidar.py

```diff
@@ -41,10 +45,25 @@ def scan(self, map: Surface) -> LaserScan:
 
             if (i != 99):
                 draw.line(map, self.red, (x_0, y_0), (x_t, y_t))
+     
+        msg = LaserScan()
+        msg.header.stamp = stamp
+        msg.header.frame_id = "laser"
+        msg.angle_min = float(0)
+        msg.angle_max = float(2*math.pi)
+        msg.angle_increment = float(2*math.pi / 60)
+        msg.time_increment = float(0)
+        msg.scan_time = float(0)
+        msg.range_min = float(0)
+        msg.range_max = float(self.range)
+        msg.ranges = []
+        msg.intensities = []
+        for point in data:
+            msg.ranges.append(math.sqrt((point[0] - x_0)**2 + (point[1] - y_0)**2)/self.m2p)
```

Comment:
Does the message include the `inf` values for the laser scans in between the scans with colision?

Reply:
Maybe we should move this calculation to the for loop above and we don't have to save the points in the `data` list